tests/test_entities.py

```diff
@@ -2204,6 +2230,53 @@ def test_creation_and_update(self):
         self.assertEqual(expected_dct, cfg_template.update_payload())
 
 
+class TemplateInputTestCase(TestCase):
+    """Tests for :class:`nailgun.entities.JobTemplate`."""
+    def setUp(self):
+        self.entity = entities.TemplateInput(config.ServerConfig('some url'), template='1')  # noqa  pylint: disable=line-too-long
+        self.read_patcher = mock.patch.object(EntityReadMixin, 'read')
```

Comment:
As TemplateInput is overriding EntityReadMixin read , perhaps we should mock the read function of TemplateInput instead of EntityReadMixin

Reply:
But then the read function of TemplateInput wouldn't be tested anymore (just like in the latest commit) and the coverage bot would fail.